server/tserver/src/test/java/org/apache/accumulo/tserver/MemValueTest.java

```diff
@@ -16,18 +16,17 @@
  */
 package org.apache.accumulo.tserver;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.junit.Assert.assertEquals;
 
-import java.nio.charset.Charset;
-
 import org.apache.accumulo.core.data.Value;
 import org.junit.Test;
 
 public class MemValueTest {
 
   @Test
   public void testDecodeDoesntModifyInputValue() {
-    Value v = new Value("2.0".getBytes(Charset.forName("UTF-8")));
+    Value v = new Value("2.0".getBytes(UTF_8));
```

Comment:
This one can just be:

```suggestion
    Value v = new Value("2.0");
```

Reply:
Actually, on second thought... I'm not so sure. This might be intentionally using a byte array to ensure the byte array isn't modified. I'm not sure what the test is trying to do. It's not entirely clear. @keith-turner what do you think?

Reply:
I think the change you suggested is find because :
 * The test does not keep a ref to the byte array for later comparison
 * The Value constructor that take a char seq will do the same thing
 * The char seq constructor for Value was not added until 1.8, so this is probably code that was written before then

Reply:
For more background 69ca25fa07b15d6d152423b9e5d15e2353837d4a may provide some insight.

Reply:
I was mostly asking whether the premise of this test (specifically that Value's underlying byte array might be mutated by the decode method) was equally valid when we're not passing in a byte array into Value's constructor. I *think* I've convinced myself that the premise is still equally valid (because the premise depends on the underlying byte array being returned by `Value.get()`, regardless of what gets passed in to the constructor).

Reply:
I'll change this to a String after this PR is merged.